tests/TestCase/Command/Phinx/SeedTest.php

```diff
@@ -133,6 +136,88 @@ public function testExecute()
         $migrations->rollback(['target' => 'all']);
     }
 
+    /**
+     * Test executing the "seed" command with bulkinsert and timestamps
+     *
+     * @return void
+     */
+    public function testExecuteTimestampFields()
```

Comment:
Did you mean to add this test to the phinx based seed command? Adding tests to `tests/TestCase/Command/SeedCommandTest.php` would be preferable.

Reply:
there were some tests already in here, but sure we could move them.

Reply:
The adapter you've changed isn't used when migrations uses the `phinx` backend which these command tests use.